extensions/aws/processors/S3Processor.cpp

```diff
@@ -143,11 +143,13 @@ minifi::utils::optional<Aws::Auth::AWSCredentials> S3Processor::getAWSCredential
 
   std::shared_ptr<core::controller::ControllerService> service = context->getControllerService(service_name);
   if (!service) {
+    logger_->log_error("AWS credentials service with name: '%s' could not be found", service_name.c_str());
```

Comment:
I think you don't need `c_str()` here.

Reply:
Updated in [48ec7a](https://github.com/apache/nifi-minifi-cpp/pull/979/commits/48ec7a2ebb6d109c147f2a3b83aa2ae8996acfa2)